tooling-docker/src/test/java/org/jboss/eap/qe/ts/common/docker/DockerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.eap.qe.ts.common.docker;
+
+import io.restassured.RestAssured;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static org.hamcrest.Matchers.containsString;
+
+public class DockerTest {
+
+    @ClassRule
+    public static Docker wildflyContainer = new Docker.Builder("wildfly", "jboss/wildfly:18.0.0.Final")
+            .setContainerReadyTimeout(60000)
+            .setContainerReadyCondition(() -> {
+                try {
+                    URL url = new URL("http://127.0.0.1:9990/health");
+                    HttpURLConnection con = (HttpURLConnection) url.openConnection();
+                    con.setRequestMethod("GET");
+                    if (con.getResponseMessage().contains("OK")) {
+                        return true;
+                    }
+                } catch (Exception ex) {
+                    // exceptions means not ready
+                    return false;
+                }
+                return false;
+            })
+            .wihtPortMapping("8080:8080")
+            .wihtPortMapping("9990:9990")
+            .withCmdArg("/opt/jboss/wildfly/bin/standalone.sh")
+            .withCmdArg("-b=0.0.0.0")
+            .withCmdArg("-bmanagement=0.0.0.0")
+            .build();
+
+    @Test
+    public void checkWelcomePage() {
+        RestAssured.when().get("http://127.0.0.1:8080").then().assertThat().body(containsString("Welcome to WildFly"));
```

Comment:
I would prefer multi-line way

Reply:
This check appears to be enough from my pov but I have no problem to add multi-line. I'll add more lines from output.

Reply:
```java
        RestAssured.when().get("http://127.0.0.1:8080").then()
                .body(containsString("Welcome to WildFly"));
```

Reply:
Another way
```java
        RestAssured.when()
                .get("http://127.0.0.1:8080").then()
                .body(containsString("Welcome to WildFly"));
```
I prefer 2 line version in this case

Reply:
Fixed.